From f2dd2691dd14880894db3d228552b2d496fc2733 Mon Sep 17 00:00:00 2001
From: Jai Luthra <j-luthra@ti.com>
Date: Thu, 19 Dec 2024 20:28:56 +0530
Subject: [PATCH 002/406] PENDING: media: cadence: csi2rx: Support RAW12 bayer
 formats

This patch adds support for RAW12 bayer formats in cadence csi2rx
driver.

Signed-off-by: Jai Luthra <j-luthra@ti.com>
Signed-off-by: Yemike Abhilash Chandra <y-abhilashchandra@ti.com>
---
 drivers/media/platform/cadence/cdns-csi2rx.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/media/platform/cadence/cdns-csi2rx.c b/drivers/media/platform/cadence/cdns-csi2rx.c
index 6f7d27a48eff..75e602c1d762 100644
--- a/drivers/media/platform/cadence/cdns-csi2rx.c
+++ b/drivers/media/platform/cadence/cdns-csi2rx.c
@@ -119,6 +119,10 @@ static const struct csi2rx_fmt formats[] = {
 	{ .code	= MEDIA_BUS_FMT_SGBRG10_1X10, .bpp = 10, },
 	{ .code	= MEDIA_BUS_FMT_SGRBG10_1X10, .bpp = 10, },
 	{ .code	= MEDIA_BUS_FMT_SRGGB10_1X10, .bpp = 10, },
+	{ .code	= MEDIA_BUS_FMT_SBGGR12_1X12, .bpp = 12, },
+	{ .code	= MEDIA_BUS_FMT_SGBRG12_1X12, .bpp = 12, },
+	{ .code	= MEDIA_BUS_FMT_SGRBG12_1X12, .bpp = 12, },
+	{ .code	= MEDIA_BUS_FMT_SRGGB12_1X12, .bpp = 12, },
 	{ .code	= MEDIA_BUS_FMT_RGB565_1X16,  .bpp = 16, },
 	{ .code	= MEDIA_BUS_FMT_RGB888_1X24,  .bpp = 24, },
 	{ .code	= MEDIA_BUS_FMT_BGR888_1X24,  .bpp = 24, },
-- 
2.39.5 (Apple Git-154)

